### PR TITLE
Storybook v7

### PR DESCRIPTION
**Pull request by stellarhoof**
Migrate to storybook v7. Most of the changes come from moving away from the `storiesOf` pattern to CSF3 (https://storybook.js.org/blog/storybook-csf3-is-here/)

### CODE REVIEW

File: packages/react/src/docs/changelog.stories.mdx
Comment (stellarhoof) [inline packages/react/src/docs/changelog.stories.mdx:1]: Couldn't find a way of importing md files

File: packages/react/src/docs/readme.stories.mdx
Comment (stellarhoof) [inline packages/react/src/docs/readme.stories.mdx:1]: Couldn't find a way of importing md files

File: packages/react/src/docs/theme.stories.js
Comment (stellarhoof) [inline packages/react/src/docs/theme.stories.js:1]: Moved to `./src/themes/Theme.stories.js`

File: packages/react/src/exampleTypes/FacetSelect.stories.js
Comment (stellarhoof) [inline packages/react/src/exampleTypes/FacetSelect.stories.js:1]: Extracted from `Facet.stories.js`

File: packages/react/src/exampleTypes/ResultTable/index.stories.js [hunk 1]
```diff
@@ -37,10 +37,33 @@ let style = (
   </style>
 )
 
-export let customizations = () => (
+export let EmojiDataset = () => {
```
Comment (stellarhoof) [inline packages/react/src/exampleTypes/ResultTable/index.stories.js:40]: Extracted from `Facet.stories.js`

File: packages/react/src/exampleTypes/TagsQuerySearchBar.stories.js
Comment (stellarhoof) [inline packages/react/src/exampleTypes/TagsQuerySearchBar.stories.js:1]: Moved from `./src/stories/basicTagsQuerySearch.js`

File: packages/react/src/queryWizard/QueryWizard.stories.js [hunk 2]
```diff
@@ -17,52 +12,20 @@ let mapNodeToDescription = (types) => (node, fields) => ({
...
-    ])}
-    title="Movies"
-  />
-)
+    ]),
+  },
+}
 
-let AccordionStory = () => (
-  <StepsAccordion>
```
Comment (stellarhoof) [inline packages/react/src/queryWizard/QueryWizard.stories.js:36]: Moved to `./src/purgatory/StepsAccordion.stories.js`

**Review by re1ro — APPROVED** [review]

**Pull request merged**